http_client/models/http_client_auth.py

```diff
@@ -13,16 +13,17 @@
 from ..value_objects import PathItem
 from .http_client_controller import _raise_endpoint_error
 
-TOKEN_AUTH_PREFIX_MAP = {'bearer': 'Bearer', 'jwt': 'JWT'}
+TOKEN_AUTH_PREFIX_MAP = {"bearer": "Bearer", "jwt": "JWT"}
```

Comment:
Please install pre-commit for this repo, so it would run expected rules, instead of your custom ones. It has a bit different pre-commit rules here. Also now its hard to see what was actually changed

And please revert these changes as it should not be part of a change (also `'` quote here is allowed and preferred over `"`)